packages/big-design/src/components/TableNext/DataCell/DataCell.tsx

```diff
@@ -9,11 +9,13 @@ export interface DataCellProps
     TableColumnDisplayProps {
   align?: 'left' | 'center' | 'right';
   children?: React.ReactNode;
+  isExpandable?: boolean;
   isCheckbox?: boolean;
   verticalAlign?: 'top' | 'middle';
   width?: number | string;
   withBorder?: boolean;
-  withPadding?: boolean;
+  withVerticalPadding?: boolean;
+  withHorizontalPadding?: boolean;
```

Comment:
Instead of defining new props, why don't we try and extend off of our `PaddingProps`? May not be possible but worth a shot.

Reply:
Tried to extended it, but things got messed up with styles, I'll leave this suggestion until the end.